### PR TITLE
Update confirm copy.

### PR DESCRIPTION
**Pull request by jmcarp**
* Factor confirm page into sub-templates
* Add missing failure and invalid data copy
* Add links to review page

### CODE REVIEW

File: regulations/templates/regulations/comment-confirm-fail.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  We're sorry, something went wrong with our system, and submitting your comment did not work.
+</div>
+
+<div>
+  <a href="{% url 'comment_review' doc_number %}">Return to your draft comment</a> to try submitting again.
+</div>
+
+<div>
+  If this problem keeps happening
+  If you've already tried submitting your comment more than once, please save a copy of your draft comment for safekeeping.
```
Comment (cmc333333) [inline regulations/templates/regulations/comment-confirm-fail.html]: Looks like a typo? I assume we don't want line 11.

Comment (jmcarp) [inline regulations/templates/regulations/comment-confirm-fail.html]: This is copied from https://docs.google.com/document/d/1Rdgvue2PG0MioLez089Dlvlqln29F1RYWi3wpRV9trk/edit?ts=571f7b84.

Comment (cmc333333) [inline regulations/templates/regulations/comment-confirm-fail.html]: It's a shame git blame will point to you :D Seriously, though, now that we've found it, let's fix it.

Comment (cmc333333) [inline regulations/templates/regulations/comment-confirm-fail.html]: Sorry, I misunderstood that it was a header in the original document but lost that formatting when you copied it. Would you mind bringing it back? Looks like we used `h3`s in the previous copy.

**Comment by jmcarp** [comment]
Updated.

**Comment by cmc333333** [comment]
Thank you kindly

**Pull request merged**